client/client_test.go

```diff
@@ -4456,7 +4456,12 @@ func testZstdLocalCacheExport(t *testing.T, sb integration.Sandbox) {
 }
 
 func testCacheExportIgnoreError(t *testing.T, sb integration.Sandbox) {
-	integration.CheckFeatureCompat(t, sb, integration.FeatureCacheExport, integration.FeatureCacheBackendLocal)
+	integration.CheckFeatureCompat(t, sb,
+		integration.FeatureCacheExport,
```

Comment:
Could we do this so that each feature is associated with the specific subtest. Atm it looks like if one backend is disabled, then tests for all backends are disabled.

Reply:
I didn't bother doing it as I was thinking disabling tests was just a temporary measure. I"m fine making this change :+1:

Reply:
I think having some cache backends disabled in moby will not be temporary.

Reply:
yes looking at the discussion about s3 and azblob that makes sense